### PR TITLE
docs: add update docs

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Upgrade docs were necessary in the case of platform resets. More detailed step-by-step instructions will be [added to the user docs](https://docs.dash.org/en/stable/masternodes/setup-testnet.html#masternode-installation) instead.

## What was done?

- Added steps to update node

## How Has This Been Tested?

tested on testnet

## Breaking Changes

none

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ $ npm install # optional: install CLI dependencies
 $ sudo npm link # optional: link CLI for system-wide execution
 ```
 
+## Update
```
Comment (shumkov) [inline README.md:47]: Let's add to TOC?

File: README.md [hunk 2]
```diff
@@ -44,6 +44,18 @@ $ npm install # optional: install CLI dependencies
...
+
+```bash
+$ mn stop
+$ git checkout master
+$ git pull
+$ mn update
+$ mn start
+```
+
+If the platform layer has been wiped, you must additionally run `$ mn reset --platform-only --hard` followed by `mn setup -k <bls-key>` with your existing BLS private key after the first `mn stop` command.
```
Comment (shumkov) [inline README.md:57]: Will the setup work if we reset the platform only?

Comment (strophy) [reply README.md]: Actually I'm not sure if `mn setup` is necessary again. Is it possible for the node to use the same `validatorKey` and `nodeKey` on the new network? Would something like this work instead?

```bash
$ mn stop
$ git checkout master
$ git pull
$ mn reset --platform-only
$ mn update
$ mn start
```

I think the other config values (`genesis`, `validators`, `contracts`) would be overwritten by the new values from the `git pull` command? Or would we need to write migrations for this?

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**